Card/card.css

```diff
@@ -0,0 +1,117 @@
+.card {
+  width: 20rem;
+
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.card-header {
+  position: relative;
+}
+
+.card-shadow {
+  border-radius: 2px;
+  box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.5);
+}
+.card-image {
+  border-radius: 3px 3px 0 0;
+  width: 100%;
+  height: auto;
+}
+.card-title {
+  font-size: 1.2rem;
+  font-weight: 600;
+}
+.card-subtitle {
+  font-size: 1.2rem;
+}
+.btn {
+  margin: 0;
+}
+
+.card-footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+}
+.card-button {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+.card-icons {
+  display: flex;
+  gap: 1rem;
+  margin-right: 0.5rem;
+}
+
+.card-icon {
+  font-size: 1.5rem;
+  color: var(--primary-color);
+
+  text-align: center;
+}
+
+/* CARD WITH TEXT OVERLAY */
+.card-title-overlay {
+  position: absolute;
+  color: white;
+  bottom: 10%;
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+/* HORIZONTAL CARD */
+.card-horizontal {
+  display: flex;
+  align-items: center;
+}
+.card-horizontal-image {
+  width: 40%;
+}
+/* CARD WITH DISMISS */
+.card-dismiss {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  right: 0;
+  z-index: 2;
+  color: white;
+  cursor: pointer;
+  height: 1.5rem;
+  width: 1.5rem;
```

Comment:
**Suggestion:** A small suggestion here is that as this class has many properties, you can sort them in ascending order alphabetically as Vikram sir had suggested. It helps in debugging or understanding the code easily.

Reply:
Hi Vishal I have added the cursor property and tried to refactor code in alphabetical order for some classes will take care of that in future .Can you please take a look and review the PR. Thanks!